cmd/bundle.go

```diff
@@ -103,32 +103,71 @@ var bundleDeprovisionCmd = &cobra.Command{
 	},
 }
 
+var bundleInitStub = &cobra.Command{
+	Use:        "init <bundle name>",
+	Deprecated: "must use 'ansible-galaxy init --type=apb <bundle name>'",
+	Hidden:     true,
+}
+
+const buildConfigCmd string = `oc new-build . --to <bundle-name>`
+const buildTriggerCmd string = `oc start-build --from-dir . <bundle-name>`
+
+var bundlePushStub = &cobra.Command{
+	Use: "push <bundle name>",
+	Deprecated: fmt.Sprintf("the OpenShift build system can be used instead.\n\n"+
+		"Create buildconfig: '%s'\n"+
+		"Start build:        '%s'\n", buildConfigCmd, buildTriggerCmd),
+	Hidden: true,
+}
+
+var bundleBuildStub = &cobra.Command{
+	Use: "build",
+	Deprecated: fmt.Sprintf("the OpenShift build system can be used instead.\n\n"+
+		"Create buildconfig: '%s'\n"+
+		"Start build:        '%s'\n", buildConfigCmd, buildTriggerCmd),
+	Hidden: true,
+}
+
 func init() {
 	bundleCmd.PersistentFlags().StringVarP(&kubeConfig, "kubeconfig", "k", "", "Path to kubeconfig to use")
 	rootCmd.AddCommand(bundleCmd)
 
 	bundlePrepareCmd.Flags().StringVarP(&bundleMetadataFilename, "bundlemeta", "b", "apb.yml", "Bundle metadata file to encode as b64")
 	bundlePrepareCmd.Flags().StringVarP(&containerMetadataFilename, "containermeta", "c", "Dockerfile", "Container metadata file to stamp")
 	bundlePrepareCmd.Flags().BoolVarP(&noLineBreaks, "nolinebreak", "n", false, "Skip adding linebreaks to b64 bundle spec")
+	rootCmd.AddCommand(createHiddenCmd(bundlePrepareCmd, "running 'apb bundle prepare'"))
 	bundleCmd.AddCommand(bundlePrepareCmd)
 
 	bundleListCmd.Flags().BoolVarP(&Refresh, "refresh", "r", false, "refresh list of specs")
+	rootCmd.AddCommand(createHiddenCmd(bundleListCmd, "running 'apb bundle list'"))
```

Comment:
Do you think its right to call `apb bundle list`? At first I did feel that was correct... but now I'm wondering if we should do `apb broker catalog` when `apb list` is called to truly get the same functionality. Thoughts?

Reply:
I think running `apb bundle list` will be most useful in our new workflow, but it will be unexpected for users of the python `apb` tool.

I'm going to add a warning to `apb list` that tells the user what happened to the old command. Maybe that will allow us to move forward without confusing prior users.